bundle/tests/scorecard/kuttl/knative2/00-runtime-knative.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: knative2-rc
+spec:
+  # Add fields here
+  applicationImage: navidsh/demo-day
+  replicas: 1
+  createKnativeService: true
```

Comment:
When knative is enabled, Deployment shouldn't exist, so we should an errors.yaml with Deployment

Reply:
I'll add that.